style/javascript.md

```diff
@@ -0,0 +1,11 @@
+# Javascript
+
+Including NodeJS and client side scripting.
+
+## [StandardJS](http://standardjs.com/)
```

Comment:
I appreciate it would make this entry very small, but do you think the linked headings are necessary? Could we simply say

> We have adopted [StandardJS](http://standardjs.com/) for all JavaScript written either in NodeJS apps or in client side scripting.
>
> Services that contain JavaScript should ensure it meets the standard